docs/get-started/quixtour/serve-sms.md

```diff
@@ -55,21 +53,19 @@ To create the SMS alert destination:
         )
 
         if response_data["messages"][0]["status"] == "0":
-            print("Message sent successfully. Admin Alerted.")
+            print("Message sent successfully.")
         else:
             print(f"Message failed with error: {response_data['messages'][0]['error-text']}")
         return
 
-
     def send_alert(row):
         """
         Trigger a CPU spike alert and send an SMS notification
         """
-        cpu_load = row['cpu_load']
-        print("Warning! CPU spike of {} detected.".format(cpu_load))
-        msg = f"Warning! CPU spike of {cpu_load} detected."
         if SEND_SMS:
-            send_sms(msg)    
+            send_sms(row["alert"]["message"])
```

Comment:
Don't even do this, just send the row. If we don't like JSON, just send a string in the alerting message. Sinks should have zero business logic.

Reply:
Changed to just send row.